### PR TITLE
Initial Finnish translations

### PR DESCRIPTION
**Pull request by kyyberi**
Translated string to Finnish, also marked none in the untranslated file

### CODE REVIEW

File: strings/strings_fi.arb [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+  "notificationFollowTitle": "Seuraaminen",
+  "notificationMentionTitle": "Maininta",
+  "notificationQuoteTitle": "Lainaus",
+  "notificationReplyTitle": "Vastaus",
+  "timerSetting": "Päivitä kaikki",
+  "quoteTo": "Lainaus {actor}",
+  "followingReplies": "Suodata vastaukset",
+  "settingsShowAlt": "Näytä alt-teksti oletusarvoisesti",
+  "confirmUnfollowActorTitle": "Lopeta seuraaminen"
+  "confirmUnfollowActorContent": "Oletko varma, että haluat lopettaa {handle} seuraamisen?",
```
Comment (myConsciousness) [inline strings/strings_fi.arb]: There is no comma in the `confirmUnfollowActorTitle` line

**Comment by myConsciousness** [comment]
Hi @kyyberi , thanks for your contribution!

There was one line that did not have a comma, so I think it can be merged if this is fixed :)

**Comment by kyyberi** [comment]
Added one missing comma and updated PR

**Comment by myConsciousness** [comment]
> Added one missing comma and updated PR

lgtm!

**Review by myConsciousness — APPROVED** [review]

**Pull request merged**